paper-server/src/main/java/org/bukkit/craftbukkit/block/CraftSkull.java

```diff
@@ -45,6 +48,7 @@ public void load(SkullBlockEntity skull) {
         if (owner != null) {
             this.profile = owner;
         }
+        this.customName = PaperAdventure.asAdventure(skull.customName);
```

Comment:
This will return an empty component when the field is null, i don't think that's what you want if the methods are nullable

Reply:
Ok, I fixed that issue, as well changed to the snapshot. I hope I did it correctly.